src/main/webapp/script.js

```diff
@@ -19,10 +20,17 @@ async function displayCodes() {
             let row = tableEl.insertRow(-1);
             var videoUrl = row.insertCell(0);
             var promoCode = row.insertCell(1);
+            var descriptionSnippet = row.insertCell(2);
             videoUrl.innerHTML = '<a href="https://www.youtube.com/watch?v='
                + codes[i].videoId +'" target="_blank">'
                + codes[i].videoTitle + '</a>';
             promoCode.innerHTML = codes[i].promoCode;
+            descriptionSnippet.innerHTML = boldSubstring(codes[i].snippet, codes[i].promoCode);
```

Comment:
Thing to note: createElement is preferred versus innerHTML when making changes to the DOM.
https://stackoverflow.com/questions/2946656/advantages-of-createelement-over-innerhtml 

We can add this later/future PR

Reply:
Ill look into that